app/docs/dev/policies/meshaccesslog.md

```diff
@@ -0,0 +1,287 @@
+---
+title: MeshAccessLog (beta)
+---
+
+With the MeshAccessLog policy you can easily set up access logs on every data plane in a mesh.
+
+{% warning %}
+This policy uses new policy matching algorithm and is in beta state,
+it should not be mixed with [TrafficLog](/docs/{{ page.version }}/policies/traffic-log).
+{% endwarning %}
+
+{% tip %}
+In the rest of this page we assume you have already configured your observability tools to work with Kuma.
+If you haven't already read the [observability docs](/docs/{{ page.version }}/explore/observability).
+{% endtip %}
+
+## TargetRef support matrix
+
+| TargetRef type    | top level | to  | from |
+|-------------------|-----------|-----|------|
+| Mesh              | ✅         | ✅   | ✅    |
+| MeshSubset        | ✅         | ❌   | ❌    |
+| MeshService       | ✅         | ✅   | ❌    |
+| MeshServiceSubset | ✅         | ❌   | ❌    |
+
+If you don't understand this table you should read [matching docs](/docs/{{ page.version }}/policies/matching).
```

Comment:
Feels like this could be a `{% %}` helper so that it's auto updated for future targetRef. Can we open an issue for this?

Reply:
https://github.com/kumahq/kuma-website/issues/1114